api/run.py

```diff
@@ -18,6 +18,8 @@
             ".",  # default, API directory
             "../packages/python/",
         ],
+        # Also reload Uvicorn when templates change.
+        reload_includes=["*.py", "*.html"],
```

Comment:
Thank you for finding this, it's really helpful for working on admin!

Reply:
💯 That is precisely why I made the PR! I got incredibly tired of `touch`-ing random `.py` files.